src/OrderFormAcceptanceTests.TestData/Models/CreateOrderItemModel.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+
+namespace OrderFormAcceptanceTests.TestData.Models
+{
+    class CreateOrderItemModel
```

Comment:
`public sealed`?

Reply:
I thought I had removed this file prior to pushing up, removed in latest commit